### PR TITLE
[New] Add collect.js@4.0.11 w/ git auto-update

### PR DESCRIPTION
**Pull request by finch185277**
Pull request for issue: #12474

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/ecrmnn/collect.js
 * Official website (optional, not the repository):
 * NPM package url (optional):https://www.npmjs.com/package/collect.js
 * License and its reference:MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:62
   - Count of stars:3019
   - Count of forks:133
 * NPM download stats (optional):
   - Downloads in the last day:226
   - Downloads in the last week:813
   - Downloads in the last month:5520

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by finch185277** [comment]
no `build` folder in `v1.0.0` -> `v4.0.0`

File: ajax/libs/collect.js/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "unique",
+    "nested-arrays",
+    "laravel-collections",
+    "comparison",
+    "collectable"
+  ],
+  "author": {
+    "name": "Daniel Eckermann",
+    "email": "danieleckermann@gmail.com",
+    "url": "http://danieleckermann.com"
```
Comment (sufuf3) [inline ajax/libs/collect.js/package.json]: Let's use `https`
Thanks

Comment (finch185277) [reply ajax/libs/collect.js/package.json]: It is not support `https`.

Comment (sufuf3) [reply ajax/libs/collect.js/package.json]: Would you try again?
https://www.danieleckermann.com/

Comment (finch185277) [reply ajax/libs/collect.js/package.json]: Thanks.

**Comment by finch185277** [comment]
no official build of v1.0.0 ~ v4.0.0

**Comment by extend1994** [comment]
@finch185277 Please remember to update PR title when auto-update config is updated next time :slightly_smiling_face:

**Pull request merged**